skypy/galaxy/_spectrum_loaders.py

```diff
@@ -94,6 +94,39 @@ def skypy_data_loader(module, name, *tags):
     return spectra
 
 
+def spectrum_template_loader(module, name, *tags):
+    '''Load kcorrect template spectrum data from the skypy module data'''
+
+    # result is spectrum or list of spectra
+    spectra = None
+
+    # load each tag separately
+    for tag in tags:
+
+        # load the data from skypy/data
+        path = astropy.utils.data._find_pkg_data_path(f'data/{module}', 'skypy')
+        filename = os.path.join(path, f'{name}_{tag}.ecsv')
```

Comment:
Would prefer not using a private Astropy function (or any Astropy function for that matter).
```suggestion
        filename = resource_filename('skypy', f'data/{module}/{name}_{tag}.ecsv')
```